README.md

```diff
@@ -26,7 +26,7 @@
 
 1. Run the command `npm install`
 
-1. Create a `.env` file with `PORT` variable
+1. Create a `.env` file with `PORT` and `DATABASE_URI` variable
 
 ### Testing the Server
 
```

Comment:
I would just make the title of this basic-api-server versus lab 3 (and so forth on other labs) if you are going to add on to the previous day lab!